howto/self-hosted-no-docker/README.md

```diff
@@ -0,0 +1,51 @@
+# Self hosted mode without Docker
+This article provides guidance on running Dapr in self-hosted mode without Docker. 
+
+## Prerequisites
+- [Dapr CLI](../../getting-started/environment-setup.md#installing-dapr-cli)
+
+## Initialize Dapr without docker
+Dapr CLI provide command line arguments to initialize Dapr [without dependency](../../getting-started/environment-setup.md#Install-Dapr-runtime-using-the-CLI-(without-docker)) on Docker. 
+
+In this mode two different binaries are installed `daprd` and `placement`. The `placement` binary is needed to enable [actors](../../concepts/actors/README.md) in Dapr. 
+
+In this mode only limited functionality of Dapr is available out of box. Namely [Service Invocation](../../concepts/service-invocation/README.md) and Actor based Service Invocation.
+
+## Service invocation
+Samples [repo](https://github.com/dapr/samples/tree/master/11.hello-dapr-slim) has a complete sample on how to perform service invocation in this mode. 
+
+## Enabling state management or pub/sub
+
+See configuring Redis in self hosted mode [without docker](../../howto/configure-redis/README.md) to enable a local state store or pub/sub broker for messaging. 
+
+## Enabling actors
+
+Run the Placement service locally to enable actor placement. Also a [transactoinal state store](#Enabling-Statestore) must be enabled for actors. 
+
+By default for Linux/MacOS the `placement` binary is installed in `/usr/local/bin` or for Windows at `c:\dapr`.
+
+```bash
+$ placement
+
+INFO[0000] starting Dapr Placement Service -- version 0.8.0 -- commit 74db927  instance=host.localhost.name scope=dapr.placement type=log ver=0.8.0
+INFO[0000] log level set to: info                        instance=host.localhost.name scope=dapr.placement type=log ver=0.8.0
+INFO[0000] metrics server started on :9090/              instance=host.localhost.name scope=dapr.metrics type=log ver=0.8.0
+INFO[0000] placement Service started on port 50005       instance=host.localhost.name scope=dapr.placement type=log ver=0.8.0
+INFO[0000] Healthz server is listening on :8080          instance=host.localhost.name scope=dapr.placement type=log ver=0.8.0
+
+```
+
+From here on you can follow the sample example created for the [java-sdk](https://github.com/dapr/java-sdk/tree/master/examples/src/main/java/io/dapr/examples/actors/http) for running an application with Actors enabled. 
```

Comment:
If this is supposed to point to Actor samples, then you should also point to the .NET and Python SDK samples.